CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ endif()
 SET ( DEFAULT_AMLOGIC                     OFF )
 SET ( DEFAULT_DISPMANX                    OFF )
 SET ( DEFAULT_OSX                         OFF )
+SET ( DEFAULT_QT                          ON  )
```

Comment:
i see you put double SET ( DEFAULT_QT                          ON  )

Reply:
Thanks for Report. 👍

Reply:
> Thanks for Report. 👍

you are welcome. And what's about BGR with Direct9X to correct byteRGB?